lib/ansible/modules/network/checkpoint/checkpoint_host.py

```diff
@@ -49,27 +49,6 @@
       - State of the access rule (present or absent). Defaults to present.
     type: str
     default: present
-  auto_publish_session:
```

Comment:
@chkp-orso since this module will be deprecated, plz remove these modules as is meaning do not change any thing in modules written by Ricky

Reply:
@justjais 
1. We already discussed it.. and I can't required those parameter because the module of ansible is relevant for add,set and delete command and there is no parameter that is required for all of those commands..
2. If I don't change Riki's modules, the tests will fail, because his modules based on some code in the modules_utils (`checkpoint`) which I deleted..